src/annotator/sidebar.js

```diff
@@ -365,7 +367,8 @@ export class Sidebar {
   _setupGestures() {
     const toggleButton = this.toolbar.sidebarToggleButton;
     if (toggleButton) {
-      this._hammerManager = new Hammer.Manager(toggleButton).on(
+      this._hammerManager = new Hammer.Manager(toggleButton);
```

Comment:
The previous code worked because `on` returns `this` (ie. it chains). However that information is missing from the types in the `@types/hammerjs` package.